content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -86,7 +86,7 @@ kubectl config --kubeconfig=config-demo set-credentials experimenter --username=
 ```
 
 {{< note >}}
-To delete a user you can run `kubectl config unset users.<name>`
+To delete a user you can run `kubectl --kubeconfig=config-demo config unset users.<name>`. To remove a cluster, you can run `kubectl --kubeconfig=config-demo config unset clusters.<name>` and to remove a context, you can run `kubectl --kubeconfig=config-demo config unset contexts.<name>`
```

Comment:
I would add an [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) here. Plus I've suggested a couple of other tweaks:
```suggestion
To delete a user, you can run `kubectl --kubeconfig=config-demo config unset users.<name>`. To remove a cluster from kubectl, you can run `kubectl --kubeconfig=config-demo config unset clusters.<name>`, and to remove a context, you can run `kubectl --kubeconfig=config-demo config unset contexts.<name>`.
```

I also agree with @kbhawkey's [suggestion](https://github.com/kubernetes/website/pull/18560#discussion_r365399001); that approach is arguably better.

Reply:
@sftim i don't think mentioning in  `To remove a cluster from kubectl` ("from kubectl") is necessary as we use kubectl before to delete users and context.Otherwise if u like to be updated like that then please let me know.
Thanks

Reply:
I can picture a user thinking that the command removes their cluster, or not being sure about the effect, and feeling that they need reassurance that wasn't in the docs. They can ask Google for extra advice for example.

If that happens, then in my view we have let that reader down.

Reply:
@sftim yes, i think describing a user in this way is good idea,i have made the changes accordingly.
Thanks